resources/kubernetes/portal-app/portal-domains.ts

```diff
@@ -1,89 +1,135 @@
 import * as k8s from '@pulumi/kubernetes';
+import * as gcp from '@pulumi/gcp';
 import { customers } from '../../get-customers';
 import { provider } from '../../shared/kubernetes/provider';
 import { debitorPortalApp } from '../debitor-portal-app/debitor-portal-app';
 import { namespace } from '../namespace';
 import { portalApi } from '../portal-api/portal-api';
 import { portalApp } from './portal-app';
+import { ingressIpAddress, zone } from '../../shared/google/dns';
+import { rootDomain } from '../../shared/config';
 
 customers.apply(customers =>
-  customers
-    .filter(c => c.domain)
-    .map(
-      customer =>
-        new k8s.networking.v1.Ingress(
-          `${customer.ident.current}-ingress`,
-          {
-            metadata: {
-              name: `customer-${customer.ident.current}`,
-              namespace: namespace.metadata.name,
-              annotations: {
-                'kubernetes.io/ingress.class': 'caddy',
+  customers.map(customer => {
+    const customerDomain = customer.domain
+      ? customer.domain
+      : `${customer.ident.current}.${rootDomain}`;
 
-                'pulumi.com/skipAwait': 'true',
-              },
-              labels: {
-                customer: customer.ident.current,
-                kind: 'customer-domain',
-              },
-            },
-            spec: {
-              rules: [
-                {
-                  host: customer.domain,
-                  http: {
-                    paths: [
-                      {
-                        path: '/',
-                        pathType: 'Prefix',
-                        backend: {
-                          service: {
-                            name: portalApp.service.metadata.name,
-                            port: { number: portalApp.port },
-                          },
-                        },
+    const domainKind = customer.domain ? 'customer-domain' : 'root-domain';
+    const debitorPortalDomain = customerDomain;
+    const creditorPortalDomain = `kred.${customerDomain}`;
```

Comment:
is `kred.{customerDomain}` the url we want to use? the pulumi var says `creditor.fpx.no`.